server.go

```diff
@@ -127,6 +128,15 @@ func (s *Stunner) Start() error {
 		}
 	}
 
+	// register metrics
```

Comment:
I guess these are safe to be run in NewStunner: the monitoring server never goes away (it's just a HTTP handler after all), so we can make a monitor.NewMonitoringBackend and do all the regs there, no?